app/views/pages/home.html.erb

```diff
@@ -1,72 +1,106 @@
+<div class="main-wrapper">
 
-<!-- Banner -->
-<div class="banner">
-  <div class="banner-content">
-      <div class="banner-head">
-        <h1>Choogle a place to go</h1>
-        <p>Decision-making tool to choose the best place to go</p>
-      </div>
-      <!-- form -->
-      <div class="date-form">
+  <div class="head">
+    <div class="head-content">
+      <h1>Where are<br>we going next ?</h1>
+      <h4>Choogle is the easiest way to collectivly choose a bar, a restaurant, an holiday destination or any other place</h4>
```

Comment:
Great! 
Just I think it's "**a** holiday" -> https://www.a-or-an.com/a_an/holiday